load-test/README.md

```diff
@@ -53,4 +53,43 @@ You can optionally pass in an iteration count for each vu to do instead of a set
 ```
 npm run benchmark --- -e VUS=#
 npm run benchmark --- -e VUS=# -e ITERS=#
+```
+
+
+## Configuring test users
+
+Oauth server supports a special test-client configuration to allow automated tools like load test generators, security scanners, to obtain a token programmatically using 'resource owner password' OIDC flow.
+
+The test client is configured only for non production environments and requires a pre provisioned client id/secret to authenticate.
+
+The following commands will create a config map from a file, attach it as a volume to the pods, and set the env var that tells the server where to find the test users data file. 
+
+```cmd
+oc -n b5e079-dev create configmap oauth-server-test-users --from-file .\test-users.json
```

Comment:
how about using something like this instead of publishing the real licenseplate here?

oc -n <repalcewith_licenseplate>-dev create configmap oauth-server-test-users --from-file .\test-users.json Thanks!